project/plugins.sbt

```diff
@@ -1,29 +1,42 @@
+// This file contains a list of SBT plugins.
+
+// === resolvers
 resolvers += Classpaths.typesafeReleases
+resolvers += Classpaths.sbtPluginReleases
+resolvers += Resolver.sonatypeRepo("releases")
+resolvers += Resolver.sonatypeRepo("snapshots")
+resolvers += "Era7 maven releases" at "http://releases.era7.com.s3.amazonaws.com"
 
-resolvers += "sonatype-releases" at "https://oss.sonatype.org/content/repositories/releases/"
+// === dependecy graph, https://github.com/jrudolph/sbt-dependency-graph (e.g. `dependencyGraphMl` command)
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.7.5")
 
+// === IntelliJ Idea project generator, https://github.com/mpeltonen/sbt-idea (e.g. `gen-idea` command)
+addSbtPlugin("com.github.mpeltonen" % "sbt-idea" % "1.7.0-SNAPSHOT")
```

Comment:
Great, thanks for hint!
http://www.scala-sbt.org/0.13/tutorial/Using-Plugins.html#Global+plugins